internal/testutil/gnmi.go

```diff
@@ -87,6 +116,12 @@ func (s *Stubber) Notification(n *gpb.Notification) *Stubber {
 	return s
 }
 
+// AppendGetResponse appends the given GetResponse as a stub response.
+func (s *Stubber) AppendGetResponse(gr *gpb.GetResponse) *Stubber {
```

Comment:
All the other methods on Stubber are named by what they stub/append and omit the word "append". Justification for this one breaking that pattern?